doc/source/analytics/logging.md

```diff
@@ -44,6 +44,26 @@ The specification is:
  * url: Any url. Optional. If not provided then it will default to the default knative borker in the namespace of the Seldon Deployment.
  * mode: Either `request`, `response` or `all`
 
+# Setting Global Default
```

Comment:
Should be `##` as this messes up the index when its top level `#`

Reply:
Ok changed